src/util/Sawyer/Assert.h

```diff
@@ -104,7 +105,7 @@ typedef void (*AssertFailureHandler)(const char *mesg, const char *expr, const s
 /** %Optional user callback to handle assertion failures.  If this variable has a non-null value, then that function is called
  *  after the failed assertion message is emitted. This allows the user to terminate the program some other way than the
  *  default @c abort call.  In any case, this function should not return; if it does, then @c abort is called. */
-extern AssertFailureHandler assertFailureHandler;
+ROSE_UTIL_API extern AssertFailureHandler assertFailureHandler;
```

Comment:
Have a look at line ~309 in src/util/Sawyer/Sawyer.h.  There's a check for ROSE_UTIL_EXPORTS here so that the rest of Sawyer doesn't rely on ROSE-specific symbols. You can change this if you need to. Just let me know so I can make the same change upstream.